src/main/resources/assets/viafabricplus/lang/en_us.json

```diff
@@ -70,25 +70,25 @@
   "authentication_settings.viafabricplus.automatically_select_cpe_when_using_classicube": "Automatically select CPE when using the ClassiCube server list.",
   "authentication_settings.viafabricplus.set_session_name_to_classicube_name": "Set session name to ClassiCube name when using the ClassiCube server list.",
 
-  "visual_settings.viafabricplus.show_super_secret_settings": "Show super secret settings",
-  "visual_settings.viafabricplus.disable_secure_chat_warning": "Disable secure chat warning",
+  "visual_settings.viafabricplus.show_super_secret_settings": "Show Super Secret Settings",
+  "visual_settings.viafabricplus.disable_secure_chat_warning": "Disable Secure Chat warning",
   "visual_settings.viafabricplus.hide_signature_indicator": "Hide signature indicator",
-  "visual_settings.viafabricplus.remove_newer_features_from_jigsaw_screen": "Remove newer features from Jigsaw screen",
-  "visual_settings.viafabricplus.replace_petrified_oak_slab": "Replace petrified oak slab with unknown texture",
-  "visual_settings.viafabricplus.emulate_armor_hud": "Emulate Armor hud",
-  "visual_settings.viafabricplus.remove_newer_features_from_command_block_screen": "Remove newer features from Command block screen",
+  "visual_settings.viafabricplus.remove_newer_features_from_jigsaw_screen": "Remove newer features from the Jigsaw screen",
+  "visual_settings.viafabricplus.replace_petrified_oak_slab": "Replace petrified oak slab texture with the 'unknown' texture",
+  "visual_settings.viafabricplus.emulate_armor_hud": "Emulate Armor HUD",
+  "visual_settings.viafabricplus.remove_newer_features_from_command_block_screen": "Remove newer features from the Commandblock screen",
   "visual_settings.viafabricplus.replace_hurt_sound_with_oof_sound": "Replace hurt sound with OOF sound",
   "visual_settings.viafabricplus.remove_newer_hud_elements": "Remove newer HUD elements",
   "visual_settings.viafabricplus.replace_creative_inventory_with_classic_inventory": "Replace creative inventory",
   "visual_settings.viafabricplus.old_walking_animation": "Old walking animation",
   "visual_settings.viafabricplus.change_font_renderer_behaviour": "Change Font renderer behaviour",
   "visual_settings.viafabricplus.enable_sword_blocking": "Enable sword animation",
-  "visual_settings.viafabricplus.enable_block_hit_animation": "Enable block hit animation",
+  "visual_settings.viafabricplus.enable_block_hit_animation": "Enable block-hit animation",
```

Comment:
No please revert. If anything it should be "Enable 'block hit' animation" but that's inconsistent with the others.

Reply:
Please revert